content/applications/websites/ecommerce/shopper_experience/payment_acquirer.rst

```diff
@@ -1,78 +1,63 @@
-
-* *Cancelled*: when the customer cancels the payment in the payment acquirer form.
-  They are taken back to Odoo in order to modify the order.
+.. image:: payment_acquirer/transaction-info.png
+   :align: center
+   :alt: page with details surrounding the specific transaction
 
 .. note:: Specific messages are provided to your customers for every
-   payment status, when they are redirected to Odoo after the transaction.
-   To edit such messages, go to the *Messages* tab of the payment
+   payment status whenever they are redirected to Odoo after the transaction.
+   To edit these messages, go to the *Messages* tab of the payment
    method.
 
+Automatically generate invoices at order
+========================================
 
-Auto-validate invoices at order
-===============================
-
-When the order is confirmed you can also have an invoice automatically issued
-and paid. This fully-automated made for businesses that invoice 
-orders straight on.
-
-.. image:: media/payment_invoice.png
-    :align: center
-
-.. note::
-   If you choose this mode you are requested to select a payment journal in order to record payments
-   in your books. This payment is automatically reconcilied with the invoice, marking it as paid.
-   Select your **bank account** if you get paid immediately on your bank account. If you don't you
-   can create a specific journal for the payment acquirer (type = Bank). That way, you can track
-   online payments in an intermediary account of your books until you get paid into your bank
-   account (see :doc:`../../../general/payment_acquirers/payment_acquirers`).
-
+When the order is confirmed, you can also choose to have an invoice automatically issued
+and paid. This fully-automated feature is designed for businesses that invoice
+orders right away.
 
-Capture the payment after the delivery
-======================================
-With this mode, the order is confirmed but the amount is kept on hold. 
-Once the delivery processed, you can capture the payment from Odoo.
-This mode is only available with Authorize.net.
+To do automatically generate invoices at order:
+  - Go to :menuselection:`Website --> Configuration --> Settings --> Invoicing`.
+  - Then, under the **Invoicing Policy** option, select *Invoice what is ordered*.
+  - Then activate *Automatic Invoices* and *Save*.
 
-.. image:: media/payment_capture_mode.png
-    :align: center
+.. image:: payment_acquirer/automatic-invoice.png
+   :align: center
+   :alt: example of automatic invoice
 
-To capture the payment, open the transaction from the order.
-Then click *Capture Transaction*.
+Capture payment after the delivery
+==================================
 
-.. image:: media/payment_capture.png
-    :align: center
+With this mode (which is only available with Authorize.net), the order is confirmed, but the amount
+is kept on hold. Once the delivery is processed, you can capture the payment from the related Sales
+Order.
```

Comment:
Although this part is correct, I'd rephrase it a bit. The "capture mode" is being added to Adyen and Stripe as we speak and I'm pretty sure that this doc won't be updated accordingly when it's done.

Instead, why not formulate it like this:
```suggestion
If the acquirer handling the payment is configured to capture amounts manually, the order is confirmed but the amount is kept on hold. Once the delivery is processed, you can capture the payment from the related Sales
Order.
```

Reply:
I had no idea the capture mode was being developed. Thanks Antoine.